src/cli/lib/printer/printer.ts

```diff
@@ -64,7 +79,9 @@ export default class Printer {
 				leftPaddingAmount = Printer.calculateLeftPadding(text);
 			}
 
+			removeLoader();
 			process.stdout.write(`${' '.repeat(leftPaddingAmount)}${formattedText}\n`);
+			printLoader();
```

Comment:
Should we add a new parameter to be able to print a message without the loader?

Reply:
Would the order of the calls mean that we can only clear the loader if we call the format function one more time?